dipy/denoise/noise_estimate.py

```diff
@@ -292,6 +331,6 @@ def estimate_sigma(arr, disable_background_masking=False):
     for i in range(sigma.size):
         convolve(arr[..., i], k, output=conv_out)
         mean_block = np.sqrt(6/7) * (arr[..., i] - 1/6 * conv_out)
-        sigma[i] = np.sqrt(np.mean(mean_block[mask]**2))
+        sigma[i] = np.sqrt(np.mean(mean_block[mask]**2) / factor)
```

Comment:
And if it wasn't bad enough, he threw that one in for good measure. I
wonder how many programmer-hours have gone to debating that one line over
the years
On May 19, 2015 6:24 PM, "Samuel St-Jean" <notifications@github.com> wrote:

> In dipy/denoise/noise_estimate.py
> <https://github.com/nipy/dipy/pull/651#discussion_r30664691>:
>
> > @@ -292,6 +331,6 @@ def estimate_sigma(arr, disable_background_masking=False):
> >      for i in range(sigma.size):
> >          convolve(arr[..., i], k, output=conv_out)
> >          mean_block = np.sqrt(6/7) * (arr[..., i] - 1/6 * conv_out)
> > -        sigma[i] = np.sqrt(np.mean(mean_block[mask]**2))
> > +        sigma[i] = np.sqrt(np.mean(mean_block[mask]**2) / factor)
>
> PEP 8:
>
> But most importantly: know when to be inconsistent -- sometimes the style
> guide just doesn't apply. When in doubt, use your best judgment.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nipy/dipy/pull/651/files#r30664691>.
>